### PR TITLE
feat: Coach data in RDS

### PR DESCRIPTION
**Pull request by alainajaffer**

### CODE REVIEW

File: packages/ref-data-retrievers/bods-retriever/index.ts [hunk 1]
```diff
@@ -79,6 +84,7 @@ export const main = async () => {
         await setupTables(dbClient);
 
         await getBodsDataAndUploadToS3(bodsUrl, txcZippedBucketName);
+        await getBodsDataAndUploadToS3(bodsUrl, txcZippedBucketName, "bodsCoach.zip");
```
Comment (ljones92) [inline packages/ref-data-retrievers/bods-retriever/index.ts]:
```suggestion
        await getBodsDataAndUploadToS3(bodsCoachUrl, txcZippedBucketName, "bodsCoach.zip");
```

File: packages/ref-data-retrievers/bods-retriever/index.ts [hunk 2]
```diff
@@ -30,7 +30,11 @@ export const setupTables = async (dbClient: Kysely<Database>) => {
     }
 };
 
-const getBodsDataAndUploadToS3 = async (bodsUrl: string, txcZippedBucketName: string) => {
+const getBodsDataAndUploadToS3 = async (
+    bodsUrl: string,
+    txcZippedBucketName: string,
+    fileName: string = "bods.zip",
```
Comment (ljones92) [inline packages/ref-data-retrievers/bods-retriever/index.ts]: I'd maybe not bother with the default here and just pass it through explicitly in the function call

File: packages/ref-data-retrievers/bods-retriever/index.ts [hunk 3]
```diff
@@ -62,14 +66,15 @@ export const main = async () => {
 
     const {
         BODS_URL: bodsUrl,
+        BODS_COACH_URL: bodCoachUrl,
```
Comment (ljones92) [inline packages/ref-data-retrievers/bods-retriever/index.ts]:
```suggestion
        BODS_COACH_URL: bodsCoachUrl,
```

File: packages/ref-data-retrievers/bods-retriever/index.ts (hunk 2 above)
Comment (alainajaffer) [reply packages/ref-data-retrievers/bods-retriever/index.ts]: done

File: packages/ref-data-retrievers/bods-retriever/index.ts (hunk 3 above)
Comment (alainajaffer) [reply packages/ref-data-retrievers/bods-retriever/index.ts]: done

File: packages/ref-data-retrievers/bods-retriever/index.ts (hunk 1 above)
Comment (alainajaffer) [reply packages/ref-data-retrievers/bods-retriever/index.ts]: done

**Review by ljones92 — DISMISSED** [review]

**Review by ljones92 — APPROVED** [review]

**Pull request merged**